ui/nuxeo-user-group-management/nuxeo-edit-user.html

```diff
@@ -80,13 +79,6 @@
           type: Boolean,
           value: false,
         },
-
-        resultFormatter: {
-          type: Function,
-          value() {
-            return this._resultFormatter.bind(this);
```

Comment:
Removing the results formatter altogether would break the API... 🤔 What about leaving the results formatter with no value so that if can be redefined?
```
        resultFormatter: {
          type: Function,
        },
```

Reply:
well considerign it is on master + this a html layout of which the purpose is to be overriden, I just thought it was cleaner and better to remove it...